env/local/sqs/terragrunt.hcl

```diff
@@ -0,0 +1,18 @@
+terraform {
+  source = "../../../aws//sqs"
+}
+
+include {
+  path = find_in_parent_folders()
+}
+
+remote_state {
+  backend = "local"
+  generate = {
+    if_exists = "overwrite_terragrunt"
+    path = "../../terraform.tfstate"
+  }
+  config = {
+    path = "../../terraform.tfstate"
+  }
+}
```

Comment:
I think you could move this repeated overwrite of the `remote_state` block up a parent level `terragrunt.hcl` file, say at `env/local/terragrunt.hcl` with this content:
```hcl
include "root" {
  path = find_in_parent_folders()
}

remote_state {
  backend = "local"
  generate = {
    if_exists = "overwrite_terragrunt"
    path = "../../terraform.tfstate"
  }
  config = {
    path = "../../terraform.tfstate"
  }
}
```

Each call to [`find_in_parent_folders()`](https://terragrunt.gruntwork.io/docs/reference/built-in-functions/#find_in_parent_folders) will return the first parent `terragrunt.hcl` file found, so you'd end up with a final config for your local stack that was built from these 3 files:
```sh
1. env/local/sqs/terragrunt.hcl 
2. env/local/terragrunt.hcl
3. env/terragrunt.hcl
```

Reply:
Thats a really good recommendation. Thanks Pat!

Reply:
Was getting an error saying you can only have one level of includes when I tried this. We can fix this after the fact